app/javascript/active_admin/features/batch_actions.js

```diff
@@ -14,7 +14,7 @@ const batchActionClick = function(event) {
     batchAction.value = this.dataset.action
   }
 
-  if (!event.target.dataset.confirm) { submitForm() }
+  if (!event.target.dataset.confirm && !event.target.dataset.modalShow) { submitForm() }
```

Comment:
We should rely on this attribute instead which will always be present for a Flowbite modal but the one used initially can be swapped out for another depending on need.

```suggestion
  if (!event.target.dataset.confirm && !event.target.dataset.modalTarget) { submitForm() }
```

Reply:
Got it! I've made the change.